integrations/slack/events/channel.go

```diff
@@ -80,6 +81,9 @@ func ChannelGroupHandler(l *zap.Logger, w http.ResponseWriter, body []byte, cb *
 		}
 	}
 
+	// Workaround for ENG-980: "is_moved: 0" should be "is_moved: false".
+	body = bytes.ReplaceAll(body, []byte(`"is_moved":0`), []byte(`"is_moved":false`))
```

Comment:
wow this is ugly. i think that instead you can write a custom UnmarshalJSON function for the struct that knows how to do this.

Reply:
I can do it, but than involved a custom bool-or-int type with a custom unmarshal function - is that really better than a single like of code?

Reply:
done